examples/utils.h

```diff
@@ -204,4 +205,30 @@ static void saveFile_orDie(const char* fileName, const void* buff, size_t buffSi
     }
 }
 
+/*! CHECK
+ * Check that the condition holds. If it doesn't print a message and die.
+ */
+#define CHECK(cond, ...)                        \
```

Comment:
`CHECK` only writes a report when a condition is not met.
It doesn't `exist()` nor `abort()` execution, which therefore proceed to next instruction.
Is that the wanted behavior ?

Reply:
Yeah thats not the intention, nice catch!